Sources/Clappr/Classes/Plugin/Container/UIContainerPlugin.swift

```diff
@@ -1,34 +1,15 @@
-open class UIContainerPlugin: UIPlugin, Plugin {
-    @objc open weak var container: Container?
-
-    open class var type: PluginType { return .container }
-
-    @objc open class var name: String {
-        return self.init().pluginName
-    }
-
-    @objc open var pluginName: String {
-        NSException(name: NSExceptionName(rawValue: "MissingPluginName"), reason: "Container Plugins should always declare a name", userInfo: nil).raise()
-        return ""
-    }
-
-    public required override init() {
-        super.init()
-    }
-
-    @objc public required init(context: UIObject) {
-        super.init()
-        if let container = context as? Container {
-            self.container = container
-        } else {
-            NSException(name: NSExceptionName(rawValue: "WrongContextType"), reason: "Container Plugins should always be initialized with a Container context", userInfo: nil).raise()
+open class UIContainerPlugin: ContainerPlugin, UIPlugin {
+    var uiObject = UIObject()
+    
+    public var view: UIView {
+        get {
+            return uiObject.view
+        } set(newValue) {
+            return uiObject.view = newValue
         }
     }
-
-    @objc open func destroy() {
-        Logger.logDebug("destroying", scope: "UIContainerPlugin")
-        Logger.logDebug("destroying listeners", scope: "UIContainerPlugin")
-        stopListening()
-        Logger.logDebug("destroyed", scope: "UIContainerPlugin")
+    
+    open func render() {
+        //NSException(name: NSExceptionName("RenderNotOverriden"), reason: "UIContainerPlugins should always override the render method").raise()
```

Comment:
Can we remove this comment?

Reply:
This is intended for the next step of the implementation.
There are tests set as `fit` to make sure every UIPlugin will override the render function.